### PR TITLE
[doc] PR for Update Openshift doc with RBAC

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3323

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md [hunk 1]
```diff
@@ -86,7 +86,9 @@ Expect the following output:
 serviceaccount/yugabyte-platform-universe-management created
 ```
 
-The next step is to grant access to this ServiceAccount using Roles and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you. To create the required RBAC objects, execute the following command:
+The next step is to grant access to this service account using Roles and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you. If you are creating clusters across multiple namespaces, you need to create Roles and RoleBindings with a cluster-admin role in each namespace where you intend to create and deploy the universe. For more information, see [RBAC resources](https://github.com/yugabyte/charts/tree/master/rbac#c-platform-namespacedyaml).
```
Comment (bhavin192) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md:89]:
```suggestion
The next step is to grant access to this service account using Roles and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you. If you are creating clusters across multiple namespaces, you need to create Roles and RoleBindings with appropriate permissions in each namespace where you intend to create and deploy the universe. For more information, see [RBAC resources](https://github.com/yugabyte/charts/tree/master/rbac#c-platform-namespacedyaml).
```
The option we are suggesting here, i.e. the namespaced one doesn't grant cluster-admin privileges.

**Review by bhavin192 — COMMENTED** [review]
The words MachineSets, ServiceAccount are names of Kubernetes/OpenShift resources, and hence they were written in that way.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md:89]: Changed "a cluster-admin role" to "appropriate permissions"

**Comment by lizayugabyte** [comment]
@baba230896 Could you review this doc PR, please?

**Comment by lizayugabyte** [comment]
@baba230896 Please let me know  you are planning to review this doc PR.

**Comment by lizayugabyte** [comment]
@baba230896 I'm going to merge this PR at 3pm EST today. If you'd like to review it, I'd appreciate if this is done by a couple hours before that time.

**Review by bhavin192 — APPROVED** [review]

**Pull request merged**